test/vonmisesfisher.jl

```diff
@@ -76,6 +76,7 @@ for (p, κ) in [(2, 1.0),
                (3, 1.0),
                (3, 5.0),
                (5, 2.0)]
+    local p
```

Comment:
Some of these might not be necessary anymore after I wrapped each file in a module from the test runner. They shouldn't hurt though...

Reply:
Always better not have unnecessary instructions, when reading the code one will wonder why these are present.

Reply:
They are just not necessary to suppress warnings. They shouldn't be confusing since they give the loop 1.0 semantics.